test/test_parse.py

```diff
@@ -71,20 +71,107 @@ def test_articles(self):
+        self.assertEqual(extract_number("this is the 3rd",
+                                        ordinals=None), 3)
+        self.assertEqual(extract_number("this is the 4th",
+                                        ordinals=None), 4)
+
+        # test non ambiguous ordinals
         self.assertEqual(extract_number("this is the first test",
                                         ordinals=True), 1)
-        self.assertEqual(extract_number("this is 2 test"), 2)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/time unit
         self.assertEqual(extract_number("this is second test",
                                         ordinals=True), 2)
-        self.assertEqual(extract_number("this is the third test"), 1.0 / 3.0)
+        self.assertEqual(extract_number("this is second test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=True), 2)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/fractional
         self.assertEqual(extract_number("this is the third test",
                                         ordinals=True), 3.0)
+        self.assertEqual(extract_number("this is the third test"),
+                         1.0 / 3.0)
+        self.assertEqual(extract_number("this is the third test",
+                                        ordinals=None), False)
+
+        self.assertEqual(extract_number("one third of a cup"), 1.0 / 3.0)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=True), 3)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=None), 1)
+
+        # test plurals
+        # NOTE plurals are never considered ordinals, but also not
+        # considered explicit fractions
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=True), 2)
+        self.assertEqual(extract_number("2 fifth",
+                                        ordinals=True), 5)
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=False), 2/5)
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=None), 2)
+
+        self.assertEqual(extract_number("Twenty two and Three Fifths"), 22.6)
+
+        # TODO fix me - see issue #152
+        # these currently return 22 instead of 3
```

Comment:
once #152 if fixed this test needs updated, tagging the issue here for reference